src/components/Campaign.tsx

```diff
@@ -53,13 +71,37 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
+                <li>
+                  {campaignContentQuery.data?.type === "unlayer" ? (
+                    <button
+                      type="button"
+                      onClick={showModal}
+                      className="dp-button button-medium primary-green"
+                    >
+                      <FormattedMessage id="save_template" />
+                    </button>
+                  ) : (
+                    ""
```

Comment:
It will be an `<li>` without content, is it fine?

Also, maybe returning `<></>` or `false` is more natural than `""`.